gitbook-docs/security.md

```diff
@@ -112,6 +126,92 @@ Clients using other kinds of protocols can include the `token` in the Signal K m
 }
 ```
 
+### Token Validation
+
+Tokens may have a short expiry time and need to be renewed periodically, or a token's current validity may be unknown. 
+
+#### Web Clients
+To validate a token, a web based client should send an HTTP GET request to `/signalk/«version»/auth/validate` with the token in the cookie.
+If the token is valid, it will be renewed, and the new cookie set in the response.
+
+The reply message will be returned in any case.
+
+#### Other Clients
+
+Clients using other kinds of protocols can send the following message.
+
+```json
+{
+  "requestId": "1234-45653-343454",
```

Comment:
No `requestId` in http responses, right?

Reply:
No, the request/response semantics return the requestId in http too (I thnk) Then the http client can check request status for long running requests that are PENDING